.cci.jenkinsfile

```diff
@@ -0,0 +1,30 @@
+@Library('github.com/coreos/coreos-ci-lib@master') _
+
+coreos.pod([image: 'registry.svc.ci.openshift.org/coreos/cosa-buildroot:latest', kvm: true, memory: "9Gi"]) {
+    checkout scm
+
+    stage("Build") {
+        coreos.shwrap("""
+        mkdir cache fcos
+        (cd fcos && coreos-assembler init https://github.com/coreos/fedora-coreos-config)
+        XDG_CACHE_HOME=$PWD/cache make install DESTDIR=fcos/overrides/rootfs
+        """)
+    }
+
+    stage("Build FCOS") {
+        coreos.shwrap("""
+        cd fcos && coreos-assembler build
+        """)
+    }
+
+    stage("Kola") {
+        try {
+            coreos.shwrap("cd fcos && cosa kola run --parallel 8")
+        } finally {
+            coreos.shwrap("tar -cf - fcos/tmp/kola/ | xz -c9 > ${env.WORKSPACE}/kola.tar.xz")
```

Comment:
`-f -` is unnecessary.

`tar c -C fcos/tmp kola` will produce a nicer tarball.

Reply:
Good call thanks! Yeah, we should probably switch to `-C ... kola` elsewhere too or centralize it in https://github.com/coreos/coreos-ci-lib.